src/cmdlib.sh

```diff
@@ -272,15 +286,26 @@ EOF
     if [ -n "${ref_is_temp}" ]; then
         echo 'ref: "'"${ref}"'"' >> "${override_manifest}"
     fi
+
     if [ -d "${configdir}/overlay" ]; then
-        echo -n "Committing ${configdir}/overlay... "
-        ostree commit --repo="${tmprepo}" --tree=dir="${configdir}/overlay" -b "${name}-config-overlay" \
-          --owner-uid 0 --owner-gid 0 --no-xattrs --no-bindings --parent=none \
-          --timestamp "${git_timestamp}"
-        cat >> "${override_manifest}" <<EOF
-ostree-layers:
-  - ${name}-config-overlay
-EOF
+        local name="${name}-config-overlay"
+        commit_overlay "${name}" "${configdir}/overlay"
+        layers="${layers} ${name}-config-overlay"
+    fi
+    if [ -d "${ovld}" ]; then
+        for n in "${ovld}"/*; do
```

Comment:
either here or in the `if [ -n ${layers} ]` section below we should sort this so conflicts are resolved in a predictable order.

Reply:
It is sorted by default: https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Filename-Expansion

> After word splitting, unless the -f option has been set (see The Set Builtin), Bash scans each word for the characters ‘*’, ‘?’, and ‘[’. If one of these characters appears, then the word is regarded as a pattern, and replaced with an alphabetically sorted list of filenames matching the pattern

Reply:
(I stated this with confidence of course, deftly hiding the fact that I googled this just before replying :wink: )